### ISSUE TITLE
Optimize Recurring Task Management to Avoid Infinite Duplication

### ISSUE THREAD

**Issue by alexandre-leng** [TechbeeAT/jtxBoard#1491]
Improvement Suggestion: Optimize Recurring Task Management to Avoid Infinite Duplication

Dear GTX Board Development Team,

I hope this message finds you well. I am writing to propose an enhancement to the recurring task management feature of the GTX Board application. Currently, when a recurring task is added, it seems to duplicate the task indefinitely

Suggestion:

Instead of duplicating the tasks indefinitely, I suggest implementing a system where the application simply reminds users at the determined repetition frequency. This would provide a cleaner user experience.

Benefits:
Improved User Experience: A cleaner and more manageable task list will enhance user satisfaction.

Thank you for considering this suggestion. Please let me know if any further details are needed.

Best regards

![Screenshot_20240629-005748](https://github.com/TechbeeAT/jtxBoard/assets/72550137/fa790d0b-adbc-4acf-a825-e5bafad55789)

**Comment by nakibrayane** [comment]
Yes, this would be so helpful, Only show a recurring tasks once, And create a new task each time the recurrence happen.

For example: when making a task that recurse every week, show only one entry the first week, and in the second week create another one, and so on.

**Comment by Mehlsine** [comment]
Three dots -> Limit recurring

**Comment by nakibrayane** [comment]
> Three dots -> Limit recurring

This resolves my issue 😅. Thank you.

**Comment by bytewright** [comment]
> Yes, this would be so helpful, Only show a recurring tasks once, And create a new task each time the recurrence happen.
> 
> For example: when making a task that recurse every week, show only one entry the first week, and in the second week create another one, and so on.

https://github.com/TechbeeAT/jtxBoard/issues/1586 My feature request would have been a workaround, seems like a similar ux issue.
